controller/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'war'
+    id 'java'
```

Comment:
Do we need `java` plugin for this module? What's about `allprojects` configuration? Please review other modules as well.

Reply:
done!